layouts/partials/header.html

```diff
@@ -36,4 +36,24 @@
             {{- end -}}
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+<div id="menu-overlay-mobile">
```

Comment:
I think we should keep only one overlay. You can hide-display the content based on the viewport width.